lib/execution_engine2/sdk/EE2Status.py

```diff
@@ -472,7 +472,9 @@ def check_jobs(
             else:
                 mongo_rec = job.to_mongo().to_dict()
                 del mongo_rec["_id"]
+                mongo_rec["retry_count"] = len(job["retry_ids"])
```

Comment:
add a returned field instead of actually adding it to the model... oh yea need to update documentation?